File: Data Structures & Algorithms/kth-largest-element-in-an-array/submission-0.cs
public class Solution {
    public int FindKthLargest(int[] nums, int k) {
        var items = new PriorityQueue<int,int>(nums.Length);
        foreach(int num in nums)
        {
            items.Enqueue(num,-num);
        }

        while(k>1)
        {
            items.Dequeue();
            k--;
        }
        return items.Peek();
    }
}

File: Data Structures & Algorithms/products-of-array-discluding-self/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number[]}
     */
    productExceptSelf(nums) {
        let map = new Map();

		for ( let i = 0; i < nums.length; i++ ) {
			let r = 1;
			for ( let j = 0; j < nums.length; j++ ) {
				if ( i === j ) {
					continue;
				} else {
					r *= nums[ j ];
				}
			}
			map.set( i, r );
		}
		return [ ...map.values() ];
    }
}
